capstone-frontend/src/components/LeftSideBar.jsx

```diff
@@ -1,44 +1,158 @@
+                  <div className="d-flex w-100 justify-content-start align-items-center">
+                    <span className="fa fa-tasks fa-fw mr-3"></span>
+                    <span id="mybooks-link" className="menu-collapsed"> My Books </span>
+                  </div>
+                </Link>
+              </Accordion.Toggle>
+              <Accordion.Collapse eventKey="2">
+                <Card.Body>My Books Dropdown</Card.Body>
+              </Accordion.Collapse>
+            </Card>
+
+            <Card>
+              <Accordion.Toggle as={Card.Header} eventKey="3">
+                <div className="bg-dark list-group-item list-group-item-action">
+                  <div className="d-flex w-100 justify-content-start align-items-center">
+                    <span className="fa fa-list fa-fw mr-3"></span>
+                    <span> My Library </span>
+                  </div>
+                </div>
+              </Accordion.Toggle>
+
+              <Accordion.Collapse eventKey="3">
+                <Card.Body id="mylists-link">
+
+                  {
+                    this.state.bookLists.map(bookList =>
+                      <div className="bg-light p-0 list-group-item list-group-item-action">
+                        <div className="d-flex w-100 justify-content-start align-items-center">
+                          <Link to={`/listpage/${bookList.id}`} key={bookList.id} className="bg-light border-0 list-group-item list-group-item-action">
+                            <span> {bookList.name}</span>
+                          </Link>
+                          <Button className="border-0 bg-transparent" onClick={() => this.deleteBookList(bookList.id)}>
+                            <svg width="1.5em" height="1.5em" viewBox="0 0 16 16" className="bi bi-trash-fill" fill="black" xmlns="http://www.w3.org/2000/svg">
+                              <path fillRule="evenodd" d="M2.5 1a1 1 0 0 0-1 1v1a1 1 0 0 0 1 1H3v9a2 2 0 0 0 2 2h6a2 2 0 0 0 2-2V4h.5a1 1 0 0 0 1-1V2a1 1 0 0 0-1-1H10a1 1 0 0 0-1-1H7a1 1 0 0 0-1 1H2.5zm3 4a.5.5 0 0 1 .5.5v7a.5.5 0 0 1-1 0v-7a.5.5 0 0 1 .5-.5zM8 5a.5.5 0 0 1 .5.5v7a.5.5 0 0 1-1 0v-7A.5.5 0 0 1 8 5zm3 .5a.5.5 0 0 0-1 0v7a.5.5 0 0 0 1 0v-7z" />
+                            </svg>
+                          </Button>
+                        </div>
+                      </div>
+
+                    )
+                  }
+
+                  <Link to="/createlist" className="bg-light list-group-item list-group-item-action">
+                    <div className="d-flex w-100 justify-content-start align-items-center">
+                      <span id="mylists-create-link"> Create New List </span>
+                    </div>
+                  </Link>
+
+                </Card.Body>
+              </Accordion.Collapse>
+            </Card>
+
+            <Card>
+              <Accordion.Toggle as={Card.Header} eventKey="4">
```

Comment:
The accordion for viewing all your lists is great! Want to add the same functionality for Clubs eventually.